OracleTuxedo/salt/samples/bankapp/Dockerfile

```
@@ -0,0 +1,20 @@
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# Pull Tuxedo base image (which also sets TUXDIR)
+# hadolint ignore=DL3007
+FROM oracle/tuxedo:latest
```

```suggestion
FROM oracle/tuxedo:22.1.0.0.0
```

Your documentation for the core Tuxedo image creates an image with a version-specific tag, so we should use that tag in downstream images too. This is to prevent this sample from possibly breaking when the next major version of Tuxedo is released.